dev-python/m2crypto/m2crypto-0.31.0-r2.ebuild

```diff
@@ -43,12 +43,17 @@ python_compile() {
 	# setup.py looks at platform.machine() to determine swig options.
 	# For exotic ABIs, we need to give swig a hint.
 	# https://bugs.gentoo.org/617946
-	# TODO: Fix cross-compiles
 	local -x SWIG_FEATURES=
 	case ${ABI} in
 		x32) SWIG_FEATURES="-D__ILP32__" ;;
 	esac
-	distutils-r1_python_compile --openssl="${EPREFIX}"/usr
+	distutils-r1_python_compile --openssl="${EROOT}"/usr
```

Comment:
EROOT is not defined outside of pkg_* phases.

Reply:
Yes, thanks, will fix it.